src/core/lib/iomgr/workqueue.h

```diff
@@ -49,34 +50,25 @@
 
 /* grpc_workqueue is forward declared in exec_ctx.h */
 
```

Comment:
It would be nice to add a small description of the `grpc_workqueue` api (i.e how to typically use `grpc_workqueue_enqueue`,  `grpc_workqueue_flush` , `grpc_workqueue_ref/unref` and `grpc_workqueue_create/destroy` functions and when does the "work" added to the workqueue gets batched and executed.